subprojects/android-test/test-report/src/test/kotlin/com/avito/android/test/report/ReportSyntheticStepsTest.kt

```diff
@@ -20,12 +22,15 @@ class ReportSyntheticStepsTest {
     private val comment = "Comment"
     private val assertionMessage = "Assertion"
 
-    @Test
-    fun `when add Entries after steps than synthetic step will be created`() {
+    @BeforeEach
+    fun before() {
         // given
         report.initTestCaseHelper()
         report.startTestCase()
+    }
 
+    @Test
+    fun `when add Entries after steps than synthetic step will be created`() {
```

Comment:
[out of scope]

We need to choose one consistent naming strategy for tests.
Examples: https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea
I see different strategies in tests. One scheme is easier to understand.

Before we were using `<state under test> <expected behaviour> <preconditions>`

`add entry after step - creates synthetic step`